packages/trader/src/sass/app/_common/form/duration.scss

```diff
@@ -0,0 +1,11 @@
+/** @define duration */
+.dc-dropdown {
```

Comment:
- Why are we separating this piece of code into its own `scss` file?
I suggest not to duplicate `dc-dropdown__display` in other files and keep it centralised in `dropdown.scss` file. 

- `.dc-dropdown__display-text` gives a left-padding to the text that needs to be removed from here to ensure the text stays at the center of the its container.

Reply:
@matin-binary 
1. we are not duplicating `dc-dropdown__display`, we have only one in `dropdown.scss` file. For duration we are using `dc-dropdown__display--duration` style. But I may add and use this style in` dropdown.scss` file. But I supposed that better don't add in file ` dropdown.scss`, which is located in package components. That’s why I add a new file in package trader.
2.  .`dc-dropdown__display-text` gives a left-padding to the text, but it's used for another case `&--is-left-text`
``` 
&--is-left-text {
            justify-content: flex-start;

            .dc-dropdown__display-text {
                padding-left: 1.2rem;
                white-space: nowrap;
                text-overflow: ellipsis;
                overflow: hidden;
                display: inline-block;
            }
        }
```

I suppose better save it and add `padding-left: none !important;` for my case:

``` .dc-dropdown {
    &__display {
        &--duration {
            border: none !important;
            .dc-dropdown__display-text {
                color: var(--text-general);
                padding-left: none !important;
            }
        }
    }
}
```
or do you have other ideas?

Reply:
1. In this case, `.dc-dropdown` and `&__display` are separated in 2 files, this is what I mean by duplicate, better to move it back to `dropdown.scss`.

2. Of course there shouldn't be any change to the current code, `padding-left: none !important;` will do the job 👍🏼

Reply:
Done